service/src/main/java/bio/terra/profile/service/profile/flight/create/CreateProfileFlight.java

```diff
@@ -48,7 +46,7 @@ public CreateProfileFlight(FlightMap inputParameters, Object applicationContext)
 
     if (CloudPlatform.AZURE == profile.cloudPlatform()) {
       // we can link the profile to the MRG only after the Sam resource has been created
-      addStep(new LinkBillingProfileIdToMrgStep(appService, samService, profile, user));
```

Comment:
`LinkBillingProfileIdToMrgStep` no longer uses any functionality that was contained in `ApplicationService` (even before the changes in this PR), so there's not any actual dependency on `ApplicationService` at all, in this flight.